### PR TITLE
updated readme with basic information

### PR DESCRIPTION
**Pull request by nadade**

### CODE REVIEW

**Comment by nadade** [comment]
resolves #2

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# pyladies-asteroids
\ No newline at end of file
+# pyladies-asteroids
+
+Code based on the Pyladies course, the Asteroids game.
```
Comment (dariagrudzien) [inline README.md]: I think it needs an apostrophe in `Asteroids`.

**Review by dariagrudzien — DISMISSED** [review]
Dude, fix it.

**Review by dariagrudzien — CHANGES_REQUESTED** [review]
Please provide a more informative commit message.

**Review by dariagrudzien — APPROVED** [review]
You won't get cookie point next time!

**Pull request merged**